crate/pages/blog/release-v0-46-4.html

```diff
@@ -0,0 +1,119 @@
+title: New release: v0.46.4
+link: https://crate.io/blog/release-v0-46-4/
```

Comment:
i'm not a SEO export, but more descriptive URLs are better, right?
(also requires to rename this file)